mssql-plugin/src/main/java/io/cdap/plugin/mssql/SqlServerDBRecord.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.mssql;
+
+import io.cdap.cdap.api.data.format.StructuredRecord;
+import io.cdap.cdap.api.data.schema.Schema;
+import io.cdap.plugin.db.DBRecord;
+import io.cdap.plugin.db.SchemaReader;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Types;
+
+import static io.cdap.plugin.mssql.SqlServerSchemaReader.GEOGRAPHY_TYPE;
+import static io.cdap.plugin.mssql.SqlServerSchemaReader.GEOMETRY_TYPE;
+
+/**
+ * Abstract writable class for SQL Server Source/Sink.
+ * It's required to have separate classes for Source and Sink in the case of separate schema readers.
```

Comment:
"in the case of separate" -> "in case of different"

Reply:
Changed.